blog/03-15-2023-developing-a-crowdfunding-platform-for-social-causes-on-celo-blockchain-part-1/developing-a-crowdfunding-platform-for-social-causes-on-celo-blockchain-part-1.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Developing a Crowdfunding Platform for Social Causes on Celo Blockchain - Part 1
+description: A Celo-based crowdfunding platform for social causes would enable social organizations, charities, and individuals to create campaigns and connect directly with donors to donate funds, leveraging the transparency and security of blockchain technology.
+authors:
+  - name: Kunal Dawar
+    title: Community Member, Celo Foundation
+    url: https://github.com/developerkunal
+    image_url: https://avatars.githubusercontent.com/u/35455566?v=4
+tags: ["intermediate", "solidity", "celosage", "truffle"]
+hide_table_of_contents: false
+slug: /tutorials/developing-a-crowdfunding-platform-for-social-causes-on-celo-blockchain-part-1
+---
+
+## Developing a Crowdfunding Platform for Social Causes on Celo Blockchain - Part 1

```

Comment:
Please remove the title from here, as it will be written automatically from the metadata